tests/UI/pages/BO/orders/add.js

```diff
@@ -797,6 +812,36 @@ class AddOrder extends BOBasePage {
     };
   }
 
+  /**
+   * Set order message
+   * @param page {Page} Browser tab
+   * @param message {string} Message text to set
+   * @returns {Promise<void>}
+   */
+  async setOrderMessage(page, message) {
+    await this.setValue(page, this.orderMessageTextArea, message);
+  }
+
+  /**
+   * Set more actions
+   * @param page {Page} Browser tab
+   * @param action {string} Action to select
+   * @returns {Promise<Page|string>}
+   */
+  // eslint-disable-next-line consistent-return
+  async setMoreActions(page, action) {
+    await this.waitForSelectorAndClick(page, this.moreActionsDropDownButton);
+    if (action === 'Send pre-filled order to the customer by email') {
+      await this.waitForSelectorAndClick(page, this.sendOrderMailButton);
+
+      return this.getTextContent(page, this.summarySuccessMessageBlock);
+    }
+    if (action === 'Proceed to checkout in the front office') {
+      const newPage = await this.openLinkWithTargetBlank(page, this.proceedOrderLink, 'body a');
+
+      return newPage;
+    }
```

Comment:
```suggestion
   return this.openLinkWithTargetBlank(page, this.proceedOrderLink, 'body a');
```
if there is only two options ou can do it like